provider/inspec.rb

```diff
@@ -56,6 +56,60 @@ def generate_resource(data)
           File.join(target_folder, "google_#{data[:product_name]}_#{name}".pluralize + '.rb')
       )
       generate_documentation(data)
+      generate_properties(data)
+    end
+
+    def generate_properties(data)
```

Comment:
What's going on with all of this?

Reply:
InSpec was using the chef/puppet way of generating properties & nested object files. This is reproducing that as it was removed.